integration-tests/utils/firecloud-utils.js

```diff
@@ -21,7 +19,19 @@ const signIntoFirecloud = async (page, token) => {
    * user-status). Instead of reworking the sign-in logic for a case that (for the most part) only
    * a computer will operate fast enough to encounter, we'll just slow the computer down a little.
    */
+  await page.waitForResponse(response => {
+    return response.url().startsWith('https://accounts.google.com/o/oauth2/') && response.request().method() === 'GET'
+  }, { timeout: 30 * 1000 }
+  )
+
+  await page.waitForXPath('//title[text()="FireCloud | Broad Institute"]')
+  await findText(page, 'content you are looking for is currently only accessible')
+  await waitForNoSpinners(page)
+
   await delay(1000) // wait a second for full load not accounted for by spinners
```

Comment:
Probably not. But I am not sure because the comment for the delay is little concerning. If it's up to me,  just leave it untouched.

Reply:
@aweng98 Could you run `test-flakes` with RUNS >= 100 for a test that uses the above logic. If they all pass, I'd say that's sufficient confidence for removing the delay.

Reply:
Ran `test-flakes` with sleep commented out. `find-workflow` test is problematic.  First time it had 26 create-workspace failures. Second time it had 5 failures on cannot find workspace link. All failures were not related to the `signIntoFirecloud` function.
I will remove sleep since non of failures is related to changes.